src/main/java/seedu/duke/command/edit/EditTaskCommand.java

```diff
@@ -1,23 +1,53 @@
 package seedu.duke.command.edit;
 
 import seedu.duke.command.CommandResult;
+import seedu.duke.data.Task;
+import seedu.duke.data.TaskManager;
 import seedu.duke.directory.Directory;
 import seedu.duke.exception.DuplicateDataException;
 import seedu.duke.exception.ModuleNotProvidedException;
+import seedu.duke.data.Task;
+import seedu.duke.data.TaskManager;
+import seedu.duke.directory.Directory;
+import seedu.duke.exception.DuplicateDataException;
+import seedu.duke.exception.TaskNotFoundException;
+import seedu.duke.util.Message;
+
+import static seedu.duke.util.ExceptionMessage.MESSAGE_DUPLICATE_TASK;
+import static seedu.duke.util.ExceptionMessage.MESSAGE_TASK_NOT_FOUND;
+import static seedu.duke.util.Message.MESSAGE_EDIT_TASK_SUCCESS;
+
 
 public class EditTaskCommand extends EditCommand {
 
-    public EditTaskCommand(int taskIndex,String newTaskDescription){
+    private int taskID;
+    private String newTaskDescription;
+    public static final String COMMAND_WORD = "edt";
```

Comment:
Is it possible to also add a EditTaskCommand format like ur EditModuleCommand

Reply:
done